app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntryEvent.kt

```diff
@@ -26,12 +26,7 @@ data class YearChanged(
 
 data class ScreenChanged(val type: ScreenType) : BloodSugarEntryEvent()
 
-data class BloodSugarSaved(val wasDateChanged: Boolean) : BloodSugarEntryEvent() {
-  override val analyticsName: String = when {
```

Comment:
Was this removed for a reason? This will no longer get pushed to analytics now. It is important for us to detect in reports if readings are being saved with the current dates or older dates.

Reply:
`wasDateChanged` was never used other than for the analytics. Should I add it back?

Reply:
Yes, if is being used for analytics, we should add it back. I think undoing this commit should be quite easy.